app/(unauthenticated)/login/page.tsx

```diff
@@ -12,20 +12,21 @@ const LoginPage = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const router = useRouter();
-  const { signIn, state } = useAuth();
+  const { login, state } = useAuth();
 
   const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setEmail(event.target.value);
+    if (event.target.value.length <= 50) setEmail(event.target.value);
   };
 
   const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setPassword(event.target.value);
+    if (event.target.value.length <= 50) setPassword(event.target.value);
   };
 
   const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
-    await signIn({ email, password });
-    if (state.isSignedIn) {
+    const success = await login({ email, password });
+    console.log(success);
```

Comment:
Can we remove this?

Reply:
Fixed [here](https://github.com/PolkadotEducation/app/pull/23/commits/7849050b418a1e9a0bf03a8d99896f049ebc124c)